docs/guides/observer.md

```diff
@@ -206,4 +208,99 @@ Next, the Loaded Intervals section displays the time intervals that have been lo
 
 The model information page concludes with a list of most frequent audits the model has failed, the most frequent time intervals that failed, and the largest historical model run times:
 
-![SQLMesh Observer historical outliers](./observer/observer_model-information-4.png)
\ No newline at end of file
+![SQLMesh Observer historical outliers](./observer/observer_model-information-4.png)
+
+## Custom measures
+
+SQLMesh Observer allows you to calculate and track custom measures in addition to the ones it [automatically calculates](#data).
+
+### Definition
+
+Each custom measure is associated with a model and is defined by a SQL query in the model file.
+
+A measure query is specified in the `@measure` macro after the primary model query. The measure query will be executed during a SQLMesh `plan` or `run` after the primary model query is executed.
+
+A measure query may return multiple columns, each of which will be stored as individual measures. Multiple measure queries may be specified in a model.
+
+This example shows a model definition that includes a measure query calculating the total number of rows in the table and the average value of the model's `numeric_col` column:
+
+```sql
+MODEL (
+    name custom_measure.example,
+    kind FULL
+);
+
+SELECT
+    numeric_col
+FROM
+    custom_measure.upstream;
+
+@measure( -- Measure query specified in the `@measure` macro
+    SELECT
+        COUNT(*) AS row_count, -- Table's row count
+        AVG(numeric_col) AS num_col_avg -- Average value of `numeric_col`
+    FROM custom_measure.example -- Select FROM the name of the model
+);
+```
+
+Every time the `custom_measure.example` model is executed, Observer will execute the measure query and store the value it returns.
+
+Observer will automatically associate the value with the execution time of the `plan`/`run` that calculated it and will record each measure under its column name. This example will create two custom measures: `row_count` and `num_col_avg`.
```

Comment:
Suggest "By default the measure's timestamp will be the execution time of the `plan`/`run` that captured the measure."